src/services/redis/routes.ts

```diff
@@ -16,7 +17,7 @@ export const RoutesCache = (): IRoutesCache => {
       await redis.set(key, value, "EX", defaultTimeToExpiryInSeconds)
       return routeToCache
     } catch (err) {
-      return new UnknownRepositoryError(err)
+      return parseRepositoryError(err)
```

Comment:
The `redis` service should not be calling anything under `mongoose/utils` I don't see why the error messages should match anyway. `redis` service needs its own parse function.

Reply:
Fixed